src/DocStringExtensions.jl

```diff
@@ -78,7 +78,7 @@ module DocStringExtensions
 export @template, FIELDS, TYPEDFIELDS, EXPORTS, METHODLIST, IMPORTS
 export SIGNATURES, TYPEDSIGNATURES, TYPEDEF, DOCSTRING, FUNCTIONNAME
 export README, LICENSE
-export interpolation
+# export interpolation
```

Comment:
@MichaelHatherly I commented out this for the release since it is a pretty generic name and doesn't seem necessary to export? We can mark it `public` in supported Julia versions later perhaps.

Reply:
Yeah, that's fine with me.